dashboard/test/ui/features/initial_page_views2.feature

```diff
@@ -6,23 +6,36 @@ Feature: Looking at a few things with Applitools Eyes - Part 2
 
   @properties_encryption_key
   Scenario Outline: Logged in simple page view without instructions dialog
+    Given I am on "http://studio.code.org/"
+    And I am a <user_type>
+    When I open my eyes to test "<test_name>"
+    And I am on "<url>"
+    And I close the instructions overlay if it exists
+    Then I see no difference for "initial load" in the current viewport
+    And I close my eyes
+    And I sign out
+    Examples:
+      | url                                        | test_name          | user_type |
+      | http://studio.code.org/projects/applab/new | new applab project | student   |
```

Comment:
I'm not certain why we wanted to specify viewport-only screenshots for this url in particular; it's possible that rather than breaking this scenario into two we could instead just update this one test to use full-page screenshots